include/seqan3/utility/type_list/traits.hpp

```diff
@@ -104,6 +104,40 @@ template <typename replace_t,
           typename ...pack_t>
 pack_traits::replace_at<replace_t, idx, pack_t...> replace_at(type_list<pack_t...>);
 
+//!\brief A helper for seqan3::list_traits::detail::unique [recursion anchor]
+template <typename query_t>
+type_list<> filter_out(type_list<>);
+
+//!\brief A helper for seqan3::list_traits::detail::unique [recursion]
+template <typename query_t, typename head_t, typename ...pack_t>
+auto filter_out(type_list<head_t, pack_t...>)
+{
+    if constexpr(std::same_as<query_t, head_t>)
+        return filter_out<query_t>(type_list<pack_t...>{});
+    else
+        return concat(type_list<head_t>{}, filter_out<query_t>(type_list<pack_t...>{}));
+}
+
+//!\brief A replacement for meta::unique [recursion anchor]
+inline constexpr type_list<> unique(type_list<>) { return {}; }
+
+//!\brief A replacement for meta::unique [recursion]
+template <typename head_t, typename ...pack_t>
+auto unique(type_list<head_t, pack_t...>)
+{
+    return concat(type_list<head_t>{}, unique(filter_out<head_t>(type_list<pack_t...>{})));
+}
+
+//!\brief A replacement for meta::reverse [recursion anchor]
+inline constexpr type_list<> reverse(type_list<>) { return {}; }
+
+//!\brief A replacement for meta::reverse [recursion]
+template <typename head_t, typename ...pack_t>
+auto reverse(type_list<head_t, pack_t...>)
+{
+    return concat(reverse(type_list<pack_t...>{}), type_list<head_t>{});
+}
+
```

Comment:
This should get proper documentation, like the other traits.
It may be a follow-up issue

Reply:
They are all in detail:: for now to skip the documentation :) but agree should be a follow-up